definitions/control-v1.yaml

```diff
@@ -4064,6 +4070,217 @@ TOfReTlUQzgpXRW5h3n2LVXbXQhPGcVitb88Cm2R8cxQwgB1VncM8yvmKhREo2tz
       - requestMode
       - source
       - target
+    ingress_postgres_outbox_rule_post:
+      type: object
+      additionalProperties: false
+      properties:
+        status:
+          type: string
+          description: The status of the rule. Rules can be enabled or disabled.
+        ruleType:
+          type: string
+          description: The type of rule. See the Ably <a href="https://ably.com/docs/general/integrations">integrations docs</a> for further information.
+          enum:
+          - ingress-postgres-outbox
+        target:
+          type: object
+          additionalProperties: false
+          properties:
+            url:
+              type: string
+              description: Postgres database connection URL.
+              example: postgres://user:password@example.com:5432/your-database-name
+            outboxTableSchema:
+              type: string
+              description: Schema for the <a href="https://ably.com/docs/livesync/outbox-nodes-tables#outbox-table">outbox table</a> in your database which allows for the reliable publication of an ordered sequence of change event messages over Ably.
+              example: public
+            outboxTableName:
+              type: string
+              description: Table name for the <a href="https://ably.com/docs/livesync/outbox-nodes-tables#outbox-table">outbox table</a>.
+              example: outbox
+            nodesTableSchema:
+              type: string
+              description: Schema for the <a href="https://ably.com/docs/livesync/outbox-nodes-tables#nodes-table">outbox table</a> in your database to allow for operation as a cluster to provide fault tolerance.
```

Comment:
"...allow for operation as a cluster..." doesn't read very well here I don't think.